plugins/woocommerce/src/Internal/Admin/RemoteFreeExtensions/DefaultFreeExtensions.php

```diff
@@ -95,6 +96,21 @@ public static function get_all() {
 			),
 		);
 
+		if ( Features::is_enabled( 'disable-core-profiler-fallback' ) ) {
+			$remote_variant_assignment = (int) get_option( 'woocommerce_remote_variant_assignment', 1 ); // 1-120
```

Comment:
Nit: Should we default to `61`, i.e: to not removing the defaults fallback? In my mind the default behaviour should be the existing behaviour, and feels safer this way